.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: File a bug report
+labels: ["Type: bug"]
+assignees: []
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug, including details regarding any error messages, version, and platform.
+      description: Please include what you expected.
+    validations:
+      required: true
+  - type: dropdown
+    id: component
+    attributes:
+      label: Component
+      description: Which component is principally affected?
+      options:
+        - Archery
+        - Benchmarking
+        - C
+        - C#
+        - C++
+        - C++ - Gandiva
+        - C++ - Plasma
+        - Compute IR
+        - Continuous Integration
+        - Dart
+        - Developer Tools
+        - Documentation
+        - FlightRPC
+        - Format
+        - GLib
+        - Go
+        - GPU
+        - Integration
+        - Java
+        - JavaScript
+        - Julia
+        - MATLAB
+        - Packaging
+        - Parquet
+        - Python
+        - R
+        - Ruby
+        - Rust
+        - Rust - Ballista
+        - Rust - DataFusion
+        - SQL
```

Comment:
What is this?

Reply:
I guess [Flight SQL](https://issues.apache.org/jira/issues/?jql=project%20%3D%20ARROW%20AND%20component%20%3D%20SQL), sort of? I'll include this in a task to reconcile the component list. This was directly copied from ASF Jira configuration.

Reply:
I don't think so, otherwise it would have been "Flight SQL" explicitly. Looking at the linked issues (if any) would be informative.

Reply:
The issues linked seem to reference Flight SQL - at least, that's how I read them.

Reply:
There would be more issues linked if it had been the case. I would vote for removing it.

Reply:
cc @lidavidm

Reply:
No, we shouldn't migrate this, we don't use it. I think it was mistakenly added. Flight SQL does not have its own component.